examples/overview-simple.py

```diff
@@ -0,0 +1,11 @@
+import pinocchio
+
+model = pinocchio.Model.BuildHumanoidSimple()
+data = model.createData()
+
+q = pinocchio.neutral(model)
+v = pinocchio.utils.zero(model.nv)
```

Comment:
I think here we should called numpy matrix directly. Just to let people aware that we can directly NumPy with Pinocchio in Python.

A simple remark, I think pinocchio.utils.matrixStuff should be made deprecated, as it is not the standard way in Python to do that. But this is only a personal viewpoint that can be discussed later.

Reply:
I think that as for now, `utils.zero` is a functionality of Pinocchio, a functionality I happen to like and use a lot because numpy's expressions can be very clumsy. If you want to move it to the main `pinocchio` namespace, why not, but it can be discussed later. I made it clear in the Overview text that we are using numpy